_posts/2021-04-08-bring-gitops-to-your-openfaas-functions-with-argocd.md

```diff
@@ -171,7 +202,9 @@ CD installation namespace.
 You can simply retrieve this password using kubectl:
 
 ```bash
-$ kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d && echo
+$ $ kubectl get secret \
```

Comment:
You have `$ $` twice here.

Also, can you wrap the line just like the output from arkade?

Then I think we can merge.

Reply:
okay, let me fix it.